hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -56,7 +56,8 @@ using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface
 // BEGIN (Handle export change): for backward compatibility
 class DummyActuator : public hardware_interface::ActuatorInterface
 {
-  CallbackReturn on_init(const hardware_interface::HardwareInfo & /*info*/) override
+  CallbackReturn on_init(
+    const hardware_interface::HardwareComponentInterfaceParams & /*info*/) override
```

Comment:
```suggestion
    const hardware_interface::HardwareComponentInterfaceParams & /*params*/) override
```
just for consistency, also for the other examples across this file